src/main/java/seedu/address/logic/parser/Parser.java

```diff
@@ -96,7 +96,7 @@ public Command parseCommand(String userInput) {
         case ListCommand.COMMAND_WORD:
             return prepareList(arguments, false);
 
-        case ListCommand.LIST_COMPLETED_COMMAND_WORD:
+        case LIST_COMPLETED_COMMAND_WORD:
```

Comment:
Quite weird that Udacity complained about using `ListCommand.LIST_COMPLETED_COMMAND_WORD` when this format is used for all other commands.